src/sentry/grouping/ingest/seer.py

```diff
@@ -217,7 +220,10 @@ def get_seer_similar_issues(
         "hash": event_hash,
         "project_id": event.project.id,
         "stacktrace": event.data.get(
-            "stacktrace_string", get_stacktrace_string(get_grouping_info_from_variants(variants))
```

Comment:
In this function, we must always prevent `request_data["stacktrace"]` to be `None`.